### PR TITLE
Add Controllers

### PR DESCRIPTION
**Pull request by devalibello**
This pull requests adds post and user controllers

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -9,4 +9,7 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
+  resources  :users do
+    resources :posts
+  end
```
Comment (ShadyShawkat) [inline config/routes.rb]: - [ ] Kindly make sure to include only the 4 routes, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Make%20sure%20that%20you%20added%20code%20to%20handle%20only%20the%204%20required%20URLs.) requirement. You can do something like the following to do so:
```ruby 
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @devalibello, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (devalibello) [reply config/routes.rb]: @ShadyShawkat Thanks for the hint on how to make my code better. The best review I have received in a long time. Keep it up 👍 😄

**Review by joel-kalema — APPROVED** [review]
Hi @devalibello 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 


## Highlights
Routes are well created 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**